keyboards/tkc/candybar/lefty_r3/readme.md

```diff
@@ -0,0 +1,15 @@
+# TheKey.Company Candybar
+
+![Candybar](https://cdn.shopify.com/s/files/1/1679/2319/products/CBR3CollectionHERO_720x.jpg)
+
+The Key Company Candybar is a staggered 40% board with a numpad utilizing the ATmega32u4 microcontroller.
+
+* Keyboard Maintainer: [Terry Mathews](https://github.com/TerryMathews/)
+* Hardware Supported: TKC Candybar
+* Hardware Availability: [TheKey.Company](https://thekey.company/products/candybar-round-3)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Added to both readme files.